src/App.tsx

```diff
@@ -12,6 +15,8 @@ function App() {
 			<div>
 				<p>ПОРУЧ ДІТИ</p>
 			</div>
```

Comment:
remove it from the main component, and make a human structure with header main and footer, where main will contain just a set of sections

Reply:
This PR has to be reviewed after previous one due to all changes to structure and so on are made there